modules/checkout/tests/src/Functional/CheckoutOrderTest.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Drupal\Tests\commerce_checkout\Functional;
+
+use Drupal\commerce_product\Entity\Product;
+use Drupal\commerce_product\Entity\ProductVariation;
+use Drupal\simpletest\BlockCreationTrait;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests the checkout of an order.
+ *
+ * @group commerce
+ *
+ * @runTestsInSeparateProcesses
```

Comment:
@runTestsInSeparateProcesses
and
@preserveGlobalState disabled
are defaults, so you don't need to specify them.